test/test_transforms.py

```diff
@@ -613,6 +559,60 @@ def test_autoaugment(self):
                 transform.__repr__()
 
 
+@pytest.mark.skipif(accimage is None, reason="accimage not available")
+class TestAccImage:
+
+    def test_accimage_to_tensor(self):
+        trans = transforms.ToTensor()
+
+        expected_output = trans(Image.open(GRACE_HOPPER).convert('RGB'))
+        output = trans(accimage.Image(GRACE_HOPPER))
+
+        torch.testing.assert_close(output, expected_output)
+
+    def test_accimage_pil_to_tensor(self):
+        trans = transforms.PILToTensor()
+
+        expected_output = trans(Image.open(GRACE_HOPPER).convert('RGB'))
+        output = trans(accimage.Image(GRACE_HOPPER))
+
+        assert expected_output.size() == output.size()
+        torch.testing.assert_close(output, expected_output, check_stride=False)
+
+    def test_accimage_resize(self):
+        trans = transforms.Compose([
+            transforms.Resize(256, interpolation=Image.LINEAR),
+            transforms.ToTensor(),
+        ])
+
+        # Checking if Compose, Resize and ToTensor can be printed as string
+        trans.__repr__()
+
+        expected_output = trans(Image.open(GRACE_HOPPER).convert('RGB'))
+        output = trans(accimage.Image(GRACE_HOPPER))
+
+        assert expected_output.size() == output.size()
+        assert np.abs((expected_output - output).mean()) < 1e-3
+        assert (expected_output - output).var() < 1e-5
+        # note the high absolute tolerance
+        self.assertTrue(np.allclose(output.numpy(), expected_output.numpy(), atol=5e-2))
```

Comment:
I think this should be         ` torch.testing.assert_close(output.numpy(), expected_output.numpy())`\\maybe you'll need to adjust atol and rtol too

Reply:
Though I've made the changes as suggested, since I don't have support for `accimage` these tests are skipped on my local machine and I'm not sure how to verify the atol and rtol values. Any ideas?

Reply:
You did well using the numpy's default for rtol. Looks like the CI doesn't even run those tests anyway... that's not good lol.

Reply:
Exactly! I had the same doubt and I checked that CI was also skipping them so I used the defaults.